server/notifications.go

```diff
@@ -17,6 +18,7 @@ func (ah *ActivityHandler) handleCreatedActivityNotification(msg *clientmodels.M
 		return metrics.DiscardedReasonChannelNotificationsUnsupported
 	}
 
+	participants := []string{}
```

Comment:
nit: `participants` and `members` sound synonymous. Maybe `notifiedUsers`?

Reply:
changed it, thanks!